core/assertions.ts

```diff
@@ -0,0 +1,39 @@
+import { isNonEmptyString, isUuidV4, UuidV4 } from './typeguards';
```

Comment:
Codacy has a fix for the issue: [Replace `'./typeguards'` with `"./typeguards"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7711626)
```suggestion
import { isNonEmptyString, isUuidV4, UuidV4 } from "./typeguards";
```